tests/test_nonbonded.py

```diff
@@ -281,47 +279,6 @@ def test_nblist_box_resize(self):
 
 
 class TestNonbonded(GradientTest):
-    def test_non_zero_du_dl(self):
-        # du_dl should be zero for this test case.
-        fp = gzip.open("tests/data/bad_test_547.pkl.gz", "rb")
-        x_t, box, lamb, nb_bp = pickle.load(fp)
-
-        nb_bp.args = nb_bp.args[:-4]  # ignore any extra args from future PRs
-
-        for i, j in nb_bp.get_exclusion_idxs():
-            assert i < j
-
-        for precision in [np.float64, np.float32]:
-
-            impl = nb_bp.unbound_impl(precision)
-            du_dx, du_dp, du_dl, u = impl.execute(x_t, nb_bp.params, box, lamb)
-
-            print(du_dx, du_dp, du_dl, u)
-
-            assert du_dl == 0.0
-
-    def test_fma_compiler_bug(self):
-
-        # this test case deals with a rather annoying fma compiler bug in CUDA.
-        # see https://github.com/proteneer/timemachine/issues/386
-        fp = gzip.open("tests/data/repro.pkl.gz", "rb")  # This assumes that primes.data is already packed with gzip
```

Comment:
This test relies on `NonbondedInterpolated` (via the pickle), and anyway the issue was seen in `du_dl`, which we don't compute anymore.